scrapper/trino/query_catalog_test.go

```diff
@@ -0,0 +1,61 @@
+package trino
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/getsynq/dwhsupport/exec/trino"
+	"github.com/stretchr/testify/suite"
+)
+
+type QueryCatalogSuite struct {
+	suite.Suite
+}
+
+func TestQueryCatalogSuite(t *testing.T) {
+	if os.Getenv("CI") != "" {
+		t.Skip("skipping Trino test in CI environment")
+	}
+	suite.Run(t, new(QueryCatalogSuite))
+}
+
+func (s *QueryCatalogSuite) TestQueryCatalog() {
+	// please run locally to make test pass:
+	// docker run --name trino -d -p 8080:8080 trinodb/trino
+	ctx := context.TODO()
+	conf := &trino.TrinoConf{
+		Host:     "localhost",
+		Port:     8080,
+		User:     "trino",
+		Password: "trino",
+		Catalogs: []string{"iceberg"},
+	}
+	scr, err := NewTrinoScrapper(ctx, conf)
+	s.Require().NoError(err)
+	s.Require().NotNil(scr)
+	defer scr.Close()
+
+	rows, err := scr.QueryCatalog(ctx)
+	s.Require().NoError(err)
+	s.Require().NotEmpty(rows)
+	spew.Dump(rows)
+
+	row := rows[0]
+	s.NotEmpty(row.Database)
+	s.NotEmpty(row.Schema)
+	s.NotEmpty(row.Table)
+	s.NotEmpty(row.Column)
+	s.NotZero(row.Position)
+	s.NotEmpty(row.Type)
+	// table_comment and comment can be nil, just check type
+	s.IsType(new(string), row.TableComment)
+	s.IsType(new(string), row.Comment)
+
+	// not supported
+	s.False(row.IsStructColumn)
+	s.False(row.IsArrayColumn)
```

Comment:
@lustefaniak `IsStructColumn` and `IsArrayColumn` is support now only for redshift by `super` type. In trino it can be different types like `postgres -> json` and so.

Also I have created this table in iceberg:
```
CREATE TABLE iceberg.default.orders (
    order_id BIGINT,
    customer_name VARCHAR,
    items ARRAY<ROW(item_id BIGINT, quantity INTEGER)>,
    created_at TIMESTAMP WITH TIME ZONE,
    metadata ROW(version VARCHAR, source VARCHAR, tags ARRAY<VARCHAR>)
);
```
and getting these types:
![CleanShot 2025-05-08 at 17 46 00@2x](https://github.com/user-attachments/assets/c4d161a4-9d42-42da-af1a-06a6500a1b76)

will it be ok?